### PR TITLE
Trigger master branch for builds

### PR DESCRIPTION
**Pull request by isacikgoz**
#### Summary
We should trigger master branch from now on.

#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Review by wiggin77 — APPROVED** [review]

**Review by crspeller — APPROVED** [review]

**Comment by isacikgoz** [comment]
Let's merge this once the GitLab MRs are merged.

**Pull request merged**
